Package.swift

```diff
@@ -19,11 +19,22 @@ let package = Package(
     targets: [
         .target(
             name: "CacheAdvance",
-            dependencies: []),
+            dependencies: ["SwiftTryCatch"],
+            swiftSettings: [.define("SWIFT_PACKAGE_MANAGER")]
```

Comment:
This enables the code to tell the difference between a build from SPM and a build from Cocoapods. In Cocoapods, `SwiftTryCatch` is not a separate module. In SPM, `SwiftTryCatch` _has_ to be a separate target (since SPM doesn't allow mixed-language targets). So I only `import SwiftTryCatch` when `SWIFT_PACKAGE_MANAGER` is set.